Binding Dapper using UrlAdaptor/Blazor WASM App/README.md

```diff
@@ -0,0 +1,17 @@
+# Blazor DataGrid Microsoft SQL Server connectivity using UrlAdaptor
+
+A project that enables data binding and CRUD action handling in the Syncfusion Blazor DataGrid to a Microsoft SQL Server using UrlAdaptor feature.
+
+## Steps to run the sample
+
+1. Download or unzip the project
+
+2. This project consist of two applications so open and run both MyWebService and GridWASM_MSSQL_UrlAdaptor projects in Visual Studio 2022
```

Comment:
project names are wrong. Correct the readme file in all the topics

Reply:
addressed the comments by correcting the project name in all the readme files.